aldryn_config.py

```diff
@@ -5,4 +5,5 @@ class Form(forms.BaseForm):
     def to_settings(self, data, settings):
         # No need to setup django-parler. That is already done in aldryn-django-cms
         settings['MIDDLEWARE_CLASSES'].insert(0, 'aldryn_redirects.middleware.RedirectFallbackMiddleware')
+        settings.setdefault('DATA_UPLOAD_MAX_NUMBER_FIELDS', 10000)
```

Comment:
where is this used?

Reply:
When User is trying to purge all the redirects from database. Sarah tried to delete all 1200 rows and got an error.

See https://docs.djangoproject.com/en/1.11/ref/settings/#data-upload-max-number-fields

Reply:
I think is best to handle this in the delete action instead.
The reason being that it feels a bit side-effecty, like you install this addon and suddenly  that global django setting changes its default (if not overriden).

Reply:
Sounds reasonable. I'll change that.